tests/Toolkit/ObjTest.php

```diff
@@ -19,6 +19,34 @@ public function test__get()
         $this->assertNull($obj->foo);
     }
 
+    public function test__getMultiple()
```

Comment:
The tests test the `get()` method, so it should be:

```suggestion
    public function testGetMultiple()
```

Reply:
@lukasbestle oh, right. I’m a dumbass. I totally saw “test__get” and thought it was just a weird notation for test get 🤦🏻‍♂️